src/cmake/cmakeEditorPanel.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright 2020 Espressif Systems (Shanghai) CO LTD
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+// limitations under the License.
+
+import { join } from "path";
+import * as vscode from "vscode";
+import { Logger } from "../logger/logger";
+import {
+  CMakeListsType,
+  loadCmakeListBuilder,
+  updateCmakeListFile,
+  updateWithValuesCMakeLists,
+} from "./cmakeListsBuilder";
+
+export class CmakeListsEditorPanel {
+  public static currentPanel: CmakeListsEditorPanel | undefined;
+
+  public static createOrShow(
```

Comment:
If already an CMake Editor Webview is open and user right-click and open another cmakelist file ... it doesn't re-render the UI.